plenum/server/primary_elector.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, node):
         # nodes. This number `1` is configurable. The reason 1 duplicate
         # message is tolerated is because sometimes when a node communicates
         # to an already lagged node, an extra NOMINATE or PRIMARY might be sent
-        self.duplicateMsgs = {}   # Dict[Tuple, int]
+        self.duplicateMsgs = {}  # Dict[Tuple, int]
```

Comment:
BTW we can remove primary elector class since it's not used.

Reply:
ok